### PR TITLE
Dt 1071

### PR DESCRIPTION
**Pull request by siren**
Adds separate data bundle for Joensuu

### CODE REVIEW

**Comment by siren** [comment]
https://digitransit.atlassian.net/browse/DT-1071

File: build-routers.sh [hunk 1]
```diff
@@ -78,6 +79,23 @@ function retrieveHsl() {
   cp hsl.zip $ROUTER_FINLAND
 }
 
+function retrieveJoensuu() {
+  echo "Retrieving Joensuu data..."
+  cd $ROUTER_JOENSUU
+  curl -sS "http://dev.hsl.fi/gtfs.waltti/joensuu.zip" -o joensuu.zip
+  curl -sS "http://dev.hsl.fi/gtfs.waltti/posjoe.zip" -o posjoe.zip
```
Comment (tuukka) [inline build-routers.sh]: Fetch these in Dockerfile instead?

**Comment by samuliheljo** [comment]
How about renaming "joensuu" -> "waltti"?

File: build-routers.sh (hunk 1 above)
Comment (siren) [inline build-routers.sh]: Did you mean that  GTFS zip retrieval for all sources should be moved to Dockerfile?

**Comment by samuliheljo** [comment]
thanks

**Pull request merged**